stdlib/public/core/StringStorage.swift

```diff
@@ -369,7 +369,7 @@ extension __StringStorage {
       __StringStorage.self,
       realCodeUnitCapacity._builtinWordValue, UInt8.self,
       1._builtinWordValue, Optional<_StringBreadcrumbs>.self)
-#if arch(i386) || arch(arm) || arch(wasm32)
+#if arch(i386) || arch(arm) || arch(wasm32) || arch(wasm32)
```

Comment:
`arch(wasm32)` condition seems to be duplicated here

Reply:
my bad 😅